bot/exts/evergreen/trivia_quiz.py

```diff
@@ -61,11 +241,13 @@ async def quiz_game(self, ctx: commands.Context, category: str = None) -> None:
 
         # Stop game if running.
         if self.game_status[ctx.channel.id] is True:
-            return await ctx.send(
-                f"Game is already running..."
+            await ctx.send(
+                "Game is already running..."
                 f"do `{self.bot.command_prefix}quiz stop`"
```

Comment:
There should be a space at the end here ("Game is already running...") so that it formats correctly in chat

Reply:
yep, that was resolved